GroupIS1024/Pages/Coaches/Create.cshtml

```diff
@@ -24,9 +24,9 @@
                 <span asp-validation-for="Coach.LastName" class="text-danger"></span>
             </div>
             <div class="form-group">
-                <label asp-for="Coach.Username" class="control-label"></label>
-                <input asp-for="Coach.Username" class="form-control" />
-                <span asp-validation-for="Coach.Username" class="text-danger"></span>
+                <label asp-for="Coach.UserName" class="control-label"></label>
+                <input asp-for="Coach.UserName" class="form-control" />
+                <span asp-validation-for="Coach.UserName" class="text-danger"></span>
```

Comment:
This same suggestion appeared in a previous code review.  When that occurs, first in wins.